blocks/product-content-block/intl.json

```diff
@@ -20,5 +20,27 @@
 		"no": "Details",
 		"pt": "Details",
 		"sv": "Details"
+	},
```

Comment:
As mentioned in the PR Dependencies, phrase translations in this file were all English. I'd appreciate assistance in the process for getting the translations. I do have access to Lokalise. Thanks!